reco_utils/dataset/sparse.py

```diff
@@ -0,0 +1,204 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""
+Generate the user/item affinity matrix from a pandas dataframe and vice versa
+"""
+
+import pandas as pd
+import numpy as np
+import itertools
+
+from scipy.sparse import coo_matrix
+import logging
+
+# import default parameters
+from reco_utils.common.constants import (
+    DEFAULT_USER_COL,
+    DEFAULT_ITEM_COL,
+    DEFAULT_RATING_COL,
+    DEFAULT_TIMESTAMP_COL,
+    PREDICTION_COL,
```

Comment:
This is not about this file but, `reco_utils.common.constants`'s `PREDICTION_COL` should be `DEFAULT_...` to align with what we have